package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
```

Comment:
```chai-http``` is in dependencies but also in dev dependencies. Can you remove it from dependencies @alainburindi

Reply:
done